benchmark-reporter.js

```diff
@@ -0,0 +1,37 @@
+
+var matchaPath = require.resolve('matcha');
+var path = require('path');
+
+var clean = require(path.join(path.dirname(matchaPath), 'lib', 'matcha', 'reporters', 'clean'));
+
+function average(list) {
+  var sum = list.reduce(function(previous, current) { return previous + current; });
+  return (sum / list.length).toFixed(0);
```

Comment:
For hypothetical other consumers, yes. For us, not unless we remove all our testcases...